src/api/table.ts

```diff
@@ -7,7 +7,7 @@ export interface UpdateVenueTableProps {
   tables: Table[];
   title: string;
   subtitle?: string;
-  capacity: number;
+  capacity?: number;
```

Comment:
Can you please explain why, this has to be optional? We already have `DEFAULT_TABLE_CAPACITY` as default value and it can be used when this is missing.

Reply:
Actually, you are right about `DEFAULT_TABLE_CAPACITY` and it's fixed in daa51bf8d3d1439f1ff1e405ab9adc44f30218d2

But the `capacity` field is optional for the `Table` type and I assume it should be aligned for the `updateVenueTable` API